courses/spl-tokens-2022-and-extensions/README.md

```diff
@@ -0,0 +1,66 @@
+# SPL Tokens: Token-2022 and Extensions
+
+## Overview
+
+This course provides an in-depth exploration of the SPL Token program on Solana, with a strong emphasis on the modern Token-2022 standard (often referred to as Token Extensions) and its advanced functionalities. The primary goal is to equip learners with the knowledge and skills to create, manage, and interact with sophisticated token features beyond basic fungible and non-fungible tokens.
+
+The course focuses on practical application, guiding students through the nuances of Token-2022 extensions such as transfer fees, interest-bearing tokens, non-transferable tokens, transfer hooks, metadata pointers, and confidential transfers. The final project involves building an NFT-Gated Content Access Program, applying these learned concepts.
```

Comment:
note: scaled ui amount is likely more useful for folks rather than interest-bearing tokens. full list is available at solana-program.com